### ISSUE TITLE
[css-backgrounds-3][css-box-3][css-color-3][css-fonts-3][css-page-4][css-speech-1][css-style-attr-1][mediaqueries-3][selectors-3][web-animations-css-integration] Convert specs to Bikeshed

### ISSUE THREAD

**Issue by SebastianZ** [w3c/csswg-drafts#251]
There are several specifications, which currently use an older preprocessor to generate the document. These specifications should be converted to use Bikeshed instead. This includes:

- [ ] [CSS Backgrounds 3](https://drafts.csswg.org/css-backgrounds-3/)
- [ ] [CSS Box 3](https://drafts.csswg.org/css-box-3/)
- [ ] [CSS Color 3](https://drafts.csswg.org/css-color-3/)
- [ ] [CSS Fonts 3](https://drafts.csswg.org/css-fonts-3/) (in transition, see issue #234)
- [ ] [CSS Paged Media 4](https://drafts.csswg.org/css-page-4/)
- [ ] [CSS Speech 1](https://drafts.csswg.org/css-speech-1/)
- [ ] [CSS Style Attributes 1](https://drafts.csswg.org/css-style-attr-1/)
- [ ] [Media Queries 3](https://drafts.csswg.org/mediaqueries-3/)
- [ ] [Selectors 3](https://drafts.csswg.org/selectors-3/)
- [ ] [Web Animations CSS Integration](https://drafts.csswg.org/web-animations-css-integration/)

*Note that I didn't include WG Notes and the CSS snapshots. If considered valuable, those should be added to the list.*

Sebastian

**Comment by tabatkins** [comment]
Box is a dead spec, so I'm removing it from the list.

**Comment by plinss** [comment]
Several of the others are RECs, so it's unclear if we want to port the drafts to Bikeshed or not at this point. If there have been significant changes the we should consider issuing a PER, but then a Bikeshed conversion may introduce too many changes? (though it would be nice for all the link/dfn markup to be consistent and functional...)

**Comment by fantasai** [comment]
The current W3C Process requires a PER even for editorial (or even markup) changes:
  https://www.w3.org/2015/Process-20150901/#rec-modify
This is a regression from the previous W3C Process:
  https://www.w3.org/2005/10/Process-20051014/tr.html#rec-modify
Analysis of the change (checked in as “editorial”) and request to revert (not addressed):
  https://lists.w3.org/Archives/Public/public-w3process/2014Oct/0143.html
Maybe Tantek can sort this out so we can update the RECs without superfluous nonsense. :)

**Comment by xfq** [comment]
The new W3C Process now allows Edited Recommendations to incorporate editorial changes without AC review: https://www.w3.org/2017/Process-20170301/#rec-modify

**Comment by svgeesus** [comment]
I don't see the value for porting Color 3. Color 4 is in bikeshed and defines everything Color 3 does plus the new stuff.

**Comment by plinss** [comment]
The main value in porting older specs is for linking. Shepherd's spec parser has some fairly effective heuristics for determining anchors types, but bikeshed generates unambiguous anchor data, so the spec will be easier (and more accurate) to link to (and easier for non-shepherd tooling to parse the spec). Second, having a bikeshed source on the draft server allows it to play in the continuous integration and keeps all links to other specs up to date. This obviously doesn't help RECs or anything on TR. Color 3 can benefit if other specs still link to it or if it links to other specs, judgement call.

**Comment by Loirooriol** [comment]
CSS Page 4 has been done in 1f0341fc70c4546d5b4ec621baae6643280b471c and 6ad91d92a0656bb1edc6edce76074deff7f6ae61.

**Comment by svgeesus** [comment]
> Color 3 can benefit if other specs still link to it 

They shouldn't; Color 4 is the official definition since [CSS Snapshot 2022](https://www.w3.org/TR/css-2022/#css-official)

**Comment by svgeesus** [comment]
For CSS Fonts 3, all of which is replaced by CSS Fonts 4, I would rather put the effort into [closing off all the Fonts 4 I18n issues](https://github.com/w3c/csswg-drafts/issues?q=is%3Aopen+label%3Ai18n-tracker+label%3Acss-fonts-4), dealing with the long-running [Fonts 4 fingerprinting issue](https://github.com/w3c/csswg-drafts/issues?q=is%3Aopen+label%3Aprivacy-tracker+label%3Acss-fonts-4) so that we can complete [horizontal review](https://github.com/w3c/csswg-drafts/issues/4944), get to CR, and update Snapshot 2023 with Fonts 4 as the official definition.

**Comment by svgeesus** [comment]
Similarly for the [Selectors 3 Rec from 2018](https://www.w3.org/TR/selectors-3/) then rather than adding errata to that or publishing an update with a bikeshed conversion, effort should focus on [Selectors 4](https://drafts.csswg.org/selectors/) which is still a WD and last published to /TR on [11 Nov 2022](https://www.w3.org/TR/2022/WD-selectors-4-20221111/)

**Comment by svgeesus** [comment]
[Web Animations CSS Integration](https://drafts.csswg.org/web-animations-css-integration/) has an ED from 2015 and has never been published to /TR so that seems pretty dead to me. Perhaps it should be retired?

**Comment by birtles** [comment]
> [Web Animations CSS Integration](https://drafts.csswg.org/web-animations-css-integration/) has an ED from 2015 and has never been published to /TR so that seems pretty dead to me. Perhaps it should be retired?

Yes, I think everything in that spec that is relevant is now incorporated in CSS Animations Level 2 and CSS Transitions Level 2.
